atlas-commerce-connector/tutorial/connecting-bigcommerce.mdx

```diff
@@ -0,0 +1,34 @@
+---
+slug: /atlas-commerce-connector/tutorial/connecting-bigcommerce
+title: Atlas Commerce Connector for BigCommerce - Connecting BigCommerce
+description: Learn how to connect BigCommerce API accounts to WordPress.
+---
+
+**Note**: These tutorials assume that you have already installed the [Atlas Commerce Blueprint via our User Portal](#)
```

Comment:
Maybe link to the first document instead?
```suggestion
**Note**: These tutorials assume that you have already installed the [Atlas Commerce Blueprint via our User Portal](introduction.mdx)
```

Reply:
That makes more sense! On the introduction page we can fill in the link to the marketing / landing / sign up material for the Blueprint / Sandbox accounts. I'll ask Bryan for that at stand up